detectron2/data/detection_utils.py

```diff
@@ -579,8 +579,13 @@ def build_augmentation(cfg, is_train):
         max_size = cfg.INPUT.MAX_SIZE_TEST
         sample_style = "choice"
     augmentation = [T.ResizeShortestEdge(min_size, max_size, sample_style)]
-    if is_train:
-        augmentation.append(T.RandomFlip())
+    if is_train and "none" not in cfg.INPUT.RANDOM_FLIP:
+        augmentation.append(
+            T.RandomFlip(
+                horizontal=True if "horizontal" in cfg.INPUT.RANDOM_FLIP else False,
+                vertical=True if "vertical" in cfg.INPUT.RANDOM_FLIP else False,
```

Comment:
```
horizontal=cfg.INPUT.RANDOM_FLIP == "horizontal",
vertical=...
```

Reply:
Same as https://github.com/facebookresearch/detectron2/pull/1976#discussion_r483426629 - accidentally left from the previous commit, should be correct now.